### ISSUE TITLE
The losses remain the same, The web doesn't learn anything

### ISSUE THREAD

**Issue by yier2333** [kvsnoufal/Pytorch-FaceNet-DogDataset#2]
I train the model with same config and dataset, but the loss is always near 0.98, same learn nothing (⊙o⊙)…

**Comment by RealYuWang** [comment]
> I train the model with same config and dataset, but the loss is always near 0.98, same learn nothing (⊙o⊙)…
same here
